bundles/framework/mydata/events/PersonaldataLoadedEvent.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
I think we can remove this event. Bundles can listen to Oskari.app `on('start')` or `on('app.start')` etc instead.